backend/src/main/java/corea/participation/controller/ParticipationControllerSpecification.java

```diff
@@ -14,6 +14,7 @@ public interface ParticipationControllerSpecification {
 
     @Operation(summary = "원하는 방에 참여합니다.",
             description = "원하는 방 상세 페이지에 위치한 참여 버튼을 누르면 참여할 수 있습니다. <br>" +
+                    "참여하는 시점에서 리뷰어로만 참여할지 / 리뷰어 & 리뷰이로 참여할 지를 결정할 수 있습니다. <br>" +
```

Comment:
```suggestion
                    "참여하는 시점에서 리뷰어로만 참여할지 / 리뷰어 & 리뷰이로 참여할지를 결정할 수 있습니다. <br>" +
```
맞춤법이 이게 맞던가요..? 저도 아리송하네요 ㅋㅋ 양쪽이 같게 수정하면 좋을 것 같다는 의도입니다!

Reply:
이게 맞네요 ㅎㅎ
맨날 헷갈리는 띄어쓰기라... 수정했습니다~! 👍